hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/request/volume/acl/TestOMVolumeSetAclRequest.java

```diff
@@ -100,7 +100,6 @@ public void testValidateAndUpdateCacheSuccess() throws Exception {
 
     OmOzoneAclMap aclMapAfterSet = omMetadataManager
         .getVolumeTable().get(volumeKey).getAclMap();
-    Assert.assertNotEquals(aclMapBeforeSet, aclMapAfterSet);
```

Comment:
Also this one.

Reply:
Because it is getting from the cache, it will have same reference.